### PR TITLE
Try to enable new pytorch version

### PR DESCRIPTION
**Pull request by eaidova**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -30,7 +30,7 @@ ppgan==2.1.*
 transformers
 
 # NNCF notebook requirements
-git+https://github.com/openvinotoolkit/nncf.git#egg=nncf[tf,torch]
+git+https://github.com/eaidova/nncf.git#egg=nncf[tf,torch]
```
Comment (paularamo) [inline requirements.txt:33]: Should we use this instead of the openvinotoolkit one?

**Review by paularamo — CHANGES_REQUESTED** [review]
Could you take a look at my question?

File: requirements.txt (hunk 1 above)
Comment (adrianboguszewski) [reply requirements.txt:33]: agree

File: requirements.txt [hunk 2]
```diff
@@ -11,12 +11,12 @@ ipywidgets
 # Upgrade onnx version to avoid 220 YOLOV5 pytorch to onnx model export issue on Windows
 onnx>=1.11.0,<1.12.0
 
-torch==1.8.1; sys_platform == 'darwin'
-torchvision==0.9.1; sys_platform == 'darwin'
+torch==1.13.0; sys_platform == 'darwin'
+torchvision==0.14.0; sys_platform == 'darwin'
```
Comment (adrianboguszewski) [inline requirements.txt:15]: I think the same should be done in Pipfile?

**Comment by eaidova** [comment]
@adrianboguszewski  @paularamo  please do not review these changes, it is draft and not ready for review, I created PR just for experiments for enabling (to see potential problem not only locally)

**Comment by adrianboguszewski** [comment]
Sorry, I saw I'm assigned and didn't notice this is a draft only.

**Comment by paularamo** [comment]
> @adrianboguszewski @paularamo please do not review these changes, it is draft and not ready for review, I created PR just for experiments for enabling (to see potential problem not only locally)

Thanks, @eaidova. Sorry, maybe I was messing things up here. In order to know when the PR is ready for review, please add these three personas once it will be ready for that: 1) technical review (your choice), 2) quality review (one member of the evangelist team), and 3) product/language review (Ryan),. Thanks again. 👍 

We could discuss those steps in further conversations. We really appreciate your contributions here.

**Pull request closed without merging**